docs/project/2024/roman.gulida/index.md

```diff
@@ -0,0 +1,97 @@
+# Cryptocurrency price tracker
+Rust-based cryptocurrency price tracker with alerts and web interface
+
+:::info 
+
+**Author**: Gulida Roman \
+**GitHub Project Link**: https://github.com/roman-gulida/project-romangulida
+
+:::
+
+## Description
+
+A cryptocurrency price tracker built with Rust on a Raspberry Pi Pico WH, featuring a screen for monitoring real-time prices and daily percentage changes. It has a switch button for toggling between different currencies and provides price alerts with a buzzer. The project also includes a web application for remote monitoring.
+
+## Motivation
+
+I've been interested in cryptocurrencies for a long time and use price tracking apps regularly. I thought it would be awesome to build my own monitoring system with a microcontroller. This project lets me create a simple tracker with alerts for buying on desired price. Plus, it's a fun way to combine my passion for crypto with hardware and coding.
+
+## Architecture 
+
+![Architecture](architecture.jpg)
+
+Components 
```

Comment:
Does this display correctly? I think all of this will be displayed in one line. Use `npm start` and test it.